src/containers/SpecialTextBoxContainer.js

```diff
@@ -0,0 +1,11 @@
+import {connect} from 'react-redux';
+import SpecialTextBox from "../components/SpecialTextBox";
+
+function mapDispatchToProps(state){
```

Comment:
This code looks like mapStateToProps, but you are suppose to be doing mapDispatchToProps